apps/sms/lib/bridge/plugins/dispose/service.js

```diff
@@ -0,0 +1,40 @@
+!((define)=>{define((require,exports,module)=>{
+
+  /**
+   * Mini Logger
+   *
+   * @type {Function}
+   */
+
+  var debug = 1 ? function(arg1, ...args) {
+    var type = `[${self.constructor.name}][${location.pathname}]`;
+    console.log(`[DisposableService]${type} - "${arg1}"`, ...args);
+  } : () => {};
+
+  module.exports = function(service) {
+    debug('attaching plugin');
+
+    service.method('@@requestDisposal', (clientId) => {
```

Comment:
Same here - it can be either server part of plugin or handled by ourselves in bridge_service_mixin.